src/services/dumpers/templates/agent-nodejs/typescript/index.hbs

```diff
@@ -0,0 +1,96 @@
+import type { Schema } from './typings';
+
+import dotenv from 'dotenv';
+import { createAgent, CollectionCustomizer } from '@forestadmin/agent';
+{{datasourceImport}}
+
+dotenv.config();
```

Comment:
That is not okey with mongoose. We import the connection before initializing the env variables. This should be placed before the datasource import.

Reply:
We might want to duplicated the env loading in the models/index.ts file as-well, let's discuss this out together when you will flush my comment 👌